src/commands/deploy.js

```diff
@@ -231,11 +32,11 @@ async function parseDeploymentOptions(cargo, config = {}, { adminAddress, confir
     },
   ]);
 
-  const flags = [
-    confirm || '--yes',
+  const flags = _.flatten([
+    confirm ? [] : ['--yes'],
     // FIXME: --dry-run is not working as expected on archwayd
-    // dryRun && '--dry-run',
-  ].filter(_.isString);
+    // dryRun ? ['--dry-run'] : [],
```

Comment:
what's missing to be fixed? once again, should we track this over an issue and add to the comment?

Reply:
I didn't investigate what's happening on `archwayd`, but I had problems running the `deploy` command with the `--dry-run` flag. I think it's worth tracking it in an issue.

Reply:
https://github.com/archway-network/archway/issues/172